src/main/java/com/openlattice/search/SearchApi.java

```diff
@@ -80,11 +78,14 @@
      * and propertyTypes are both specified, the propertyTypes param will be ignored.
      *
      * @param search A JSON object that contains between three and five parameters. Required parameters are "start" and
-     *               "maxHits, which specify the hit number to start returning results on for paging and the maximum number
+     *               "maxHits, which specify the hit number to start returning results on for paging and the maximum
+     *               number
```

Comment:
super minor, but can you format this so the lines are the full length (or line breaks are at logical places)?